ppr-ui/tests/unit/RegistrationLengthTrust.spec.ts

```diff
@@ -204,6 +204,16 @@ describe('RegistrationLengthTrust SA renewal test', () => {
     expect(wrapper.find('#new-expiry').text()).toContain('March 31, 2022')
     expect(wrapper.vm.$store.state.stateModel.registration.lengthTrust.lifeYears).toBe(1)
     expect(wrapper.vm.$store.state.stateModel.registration.lengthTrust.valid).toBe(true)
+    //also emits if valid
+    expect(wrapper.emitted().lengthTrustValid).toBeTruthy()
```

Comment:
hey steph can you use the `getLastEvent` fn in utils instead for before + after for this test? Checking emitted only checks that the wrapper emitted some value for `lengthTrustValid` at some point since it was rendered so it doesn't really tell us much